### DISCUSSION TITLE
谷歌服务已对该IP地址进行限流

### DISCUSSION THREAD

**Discussion by feng-kui** [immersive-translate/immersive-translate#1929] [Help]
### 概述 | Summary

无法翻译，显示限流。但尝试过更换🪜节点，没有效果。打开谷歌翻译，是正常的
<img width="294" alt="image" src="https://github.com/user-attachments/assets/a3bfdd9e-c0a1-4069-9a7e-682c37758524">

**Comment by lanceyliao** [comment]
同遇到此问题

**Comment by keyboard3** [comment]
https://github.com/immersive-translate/immersive-translate/issues/2547 详情可以看下这里

**Comment by lanceyliao** [comment]
能提供下api的url或者域名吗？我可以用负载均衡（轮询load-balance round-robin 把所有的请求分配给策略组内不同的代理节点）访问google
https://wiki.metacubex.one/config/proxy-groups/load-balance/#strategy

**Reply by keyboard3** [reply]
现在是谷歌对所有节点都限流，没办法

**Reply by lanceyliao** [reply]
不是，比如我有100条要翻译，但我有100个节点，负载均衡轮询是每个节点请求1次翻译，不会触发限流吧

**Reply by keyboard3** [reply]
![image](https://github.com/user-attachments/assets/c0fffe5b-2274-497f-8845-2aa101113ebe)
强行开启，然后抓包吧
```
  "translationServices": {
    "google": {
      "enableBrowserApi": true
    }
  },
```

**Comment by lanceyliao** [comment]
translate.googleapis.com
translate-pa.googleapis.com
